src/providers/TokensProvider.tsx

```diff
@@ -58,6 +60,14 @@ export default function TokensProvider({
           targetTokenAddresses.add(address)
         })
       }
+      if (gauges.gauges) {
+        // add gauge tokens
+        ;(Object.values(gauges.gauges) as Gauge[]).forEach((gauge) => {
+          gauge.rewards.forEach(({ token }) => {
```

Comment:
minichefData pulls out all reward tokens into their own array at the top level of the object. I might want to do that here, but I don't think it makes much of a difference.